RSSReader/Services/RSSParser.h

```diff
@@ -0,0 +1,24 @@
+//
+//  RSSParser.h
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 19.11.20.
+//
+
+#import <Foundation/Foundation.h>
+#import "FeedItem.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RSSParser : NSObject <NSXMLParserDelegate>
+
+@property (atomic, retain)NSMutableArray<FeedItem *>* array;
+@property (atomic, retain)NSString *currentElement;
+
+- (instancetype)init UNAVAILABLE_ATTRIBUTE;
++ (instancetype)sharedInstance;
+- (void)parseFeedWithData:(NSData *)data andArray:(NSMutableArray<FeedItem *>*)array completion:(void(^)(BOOL success))completion;
```

Comment:
make array as an object to return from completion, rely only on references mechanism is a bad idea, IMO